src/connector/src/parser/csv_parser.rs

```diff
@@ -31,6 +31,20 @@ macro_rules! parse {
     };
 }
 
+/// Parse a string into a boolean value, following Go's `strconv.ParseBool`.
+fn parse_boolean(s: &str) -> AccessResult<bool> {
```

Comment:
What about following PostgreSQL `boolin` rather than Go `strconv.ParseBool`? The former is already defined in `common`:
https://github.com/risingwavelabs/risingwave/blob/61e9e526b675ce8dc9e878324e1b7d374f72ab03/src/common/src/cast/mod.rs#L23-L31

Reply:
Oh off course, I didn't find it.

Reply:
Fixed